### PR TITLE
feat: add GetBytes method and test cases

### PR DESCRIPTION
**Pull request by devhaozi**
Closes goravel/goravel#234

## 📑 Description

Add GetBytes method and test cases

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

File: filesystem/local.go [hunk 1]
```diff
@@ -143,6 +143,15 @@ func (r *Local) Get(file string) (string, error) {
 	return string(data), nil
 }
 
+func (r *Local) GetBytes(file string) ([]byte, error) {
```
Comment (hwbrzzl) [inline filesystem/local.go:146]: S: The `Get` method can call this method.

**Comment by devhaozi** [comment]
Seems we can remove the workflow file and the workflow secrets for filesystem because all the drivers was move to their own repository.

cc @hwbrzzl

File: .github/workflows/filesystem.yml
Comment (hwbrzzl) [inline .github/workflows/filesystem.yml:1]: Good catch

File: filesystem/local_test.go
Comment (hwbrzzl) [inline filesystem/local_test.go:1]: S: We can move the test cases in `application_test.go` to here.

**Review by hwbrzzl — COMMENTED** [review]
Great work!

File: filesystem/local.go (hunk 1 above)
Comment (devhaozi) [reply filesystem/local.go:146]: > S: The `Get` method can call this method.

I will optimize this.

File: filesystem/local_test.go [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+	s.Equal([]string{"3/", "3/4/", "3/5/", "3/5/6/"}, files)
+	files, err = s.local.AllDirectories("./AllDirectories")
+	s.Nil(err)
+	s.Equal([]string{"3/", "3/4/", "3/5/", "3/5/6/"}, files)
+	files, err = s.local.AllDirectories("/AllDirectories")
+	s.Nil(err)
+	s.Equal([]string{"3/", "3/4/", "3/5/", "3/5/6/"}, files)
+	files, err = s.local.AllDirectories("./AllDirectories/")
+	s.Nil(err)
+	s.Equal([]string{"3/", "3/4/", "3/5/", "3/5/6/"}, files)
```
[the remark this answers is not in the document]
Comment (hwbrzzl) [reply filesystem/local_test.go:81]: Can we use `runtime.GOOS` to judge the Windows system and set a different assertion?

File: .github/workflows/codecov.yml [hunk 3]
```diff
@@ -12,12 +12,12 @@ jobs:
...
-      - name: Install dependencies 📦
+      - name: Install dependencies
         run: go mod tidy
-      - name: Run tests with coverage ✅
+      - name: Run tests with coverage
         run: go test -v -coverprofile="coverage.out" ./...
-      - name: Upload coverage report to Codecov 📝
-        uses: codecov/codecov-action@v4
+      - name: Upload coverage report to Codecov
+        uses: codecov/codecov-action@v3
```
Comment (devhaozi) [inline .github/workflows/codecov.yml:20]: Seems codecov delete their v4 release. So I fallback it to v3.

Comment (hwbrzzl) [reply .github/workflows/codecov.yml:20]: So bad.

File: filesystem/local_test.go [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+
+import (
+	"context"
+	"mime"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/goravel/framework/support/carbon"
```
Comment (hwbrzzl) [inline filesystem/local_test.go:11]: S: Order import

**Review by hwbrzzl — APPROVED** [review]
LGTM

**Pull request merged**